tests/RobotFramework/tasks/E2E_Tutorials/raising_alarms.robot

```diff
@@ -0,0 +1,96 @@
+*** Settings ***
+
+Resource              ../../../../resources/common.resource
+Library               ThinEdgeIO    adapter=${ADAPTER}
+Library               Cumulocity
+Library               String
+Suite Setup           Custom Setup
+Suite Teardown        Custom Teardown
+
+*** Variables ***
+
+${ADAPTER}            ssh
+${C8Y_ROOT_CERT_PATH}    /etc/ssl/certs
+
+*** Tasks ***
+
+Raise Alarm
+    [Documentation]    This test case raises an alarm using ThinEdgeIO. 
+    ...                It publishes an alarm message with specified type, 
+    ...                text, and severity to the device and verifies the alarm's presence.
+    Raise Alarm Keyword    Current_high    Current is high    critical
+
+Raise Alarm With Timestamp
+    [Documentation]    This test case raises an alarm with a specific timestamp using ThinEdgeIO. 
+    ...                It publishes an alarm message with specified type, text, severity, 
+    ...                and timestamp to the device and verifies the alarm's presence.
+    Raise Alarm With Timestamp Keyword    temperature_high    Temperature is very high    critical   2021-01-01T05:30:45+00:00 
+
+Raise Custom Alarm
+    [Documentation]    This test case raises a custom alarm using Cumulocity. 
+    ...                It publishes an alarm message with specified type, text, 
+    ...                severity, and a custom fragment to the device and verifies the alarm's presence.
+    Raise Custom Alarm Keyword    PIR    Person detected    critical    someOtherCustomFragment
+    Raise Custom Alarm Keyword    PIR    Person detected    critical    someOtherCustomFragment
+
+*** Keywords ***
+
+Raise Alarm Keyword
+    [Documentation]    Publishes an alarm message with the specified type, text, 
+    ...                and severity to the device using ThinEdgeIO MQTT. 
+    ...                Verifies that the alarm is present on the device.
+    [Arguments]    ${alarm_type}    ${text}    ${severity}
+    ${payload}=    Set Variable    "text": "${text}", "severity": "${severity}"
+    Execute Command    tedge mqtt pub te/device/main///a/${alarm_type} '{${payload}}' -r -q 2
```

Comment:
Slight refactoring to make it more readable (and it can be applied to all of the other keywords which use a similar setup)

```suggestion
    ${payload}=    Set Variable    {"text": "${text}", "severity": "${severity}"}
    Execute Command    tedge mqtt pub te/device/main///a/${alarm_type} '${payload}' -r -q 2
```